Common/HttpWebRequestWrapperLib/HttpWebRequestWrapper.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Text;
+
+namespace HttpWebRequestWrapperLib
+{
+    /// <summary>
+    /// Wrapper of http web requests and responses that encapsulate GET, PUT POST and DELETE requests
+    /// </summary>
+    public class HttpWebRequestWrapper
+    {
+        public HttpWebRequest httpWebRequest { get; set; }
+        public HttpWebResponse httpWebResponse { get; set; }
+
+        public HttpWebRequestWrapper(string contentType = "application/json")
+        {
+            httpWebRequest.ContentType = contentType;
+        }
+        
+        public string Get(string url, Dictionary<string,string> parameters)
+        {
+            if (string.IsNullOrEmpty(url))
+            {
+                return "Url is null or empty";
+            }
+            if (!(parameters is null) || parameters.Count > 0)
+            {
+                url = getUrlWithParams(url, parameters);
+            }
```

Comment:
May be, it have reason to take out these validations in special methods?

Reply:
Hm, it isn't validation, actually. But I understand your point: but let's see, whether else classes and methods will use this function? If all logic with requests/responses will be in this class?